### ISSUE TITLE
Nuget package PCL version

### ISSUE THREAD

**Issue by jcdekoning** [NetTopologySuite/GeoAPI#7]
Hello,

When will the PCL version of the GeoAPI be available as nuget package?

Johan

**Comment by pauldendulk** [comment]
There is already a pcl version in the regular GeoAPI package. Profile147: portable-net403+sl5+win8+wp8+MonoAndroid+MonoTouch

**Comment by FObermaier** [comment]
Have a look at https://github.com/NetTopologySuite/NetTopologySuite/issues/39

**Comment by jcdekoning** [comment]
When I did nuget install-package I got this error 

install-package : Could not install package 'GeoAPI 1.7.3.1'. You are trying to install this package into a project that targets 'portable-net45+sl50+win+wpa81+wp80', but the package does not contain any assembly references or c
ontent files that are compatible with that framework. For more information, contact the package author.

Probably a profile conflict. GeoAPI is using Profile 147 (.NET Framework 4.0.3, Silverlight 5, Windows 8, Windows Phone Silverlight 8) and the other project is Profile 344 (.NET Framework 4.5, Silverlight 5, Windows 8, Windows Phone 8.1, Windows Phone Silverlight 8)
